crates/meilisearch/tests/dumps/mod.rs

```diff
@@ -2380,6 +2380,7 @@ async fn generate_and_import_dump_containing_vectors() {
           "source": "huggingFace",
           "model": "sentence-transformers/all-MiniLM-L6-v2",
           "revision": "e4ce9877abf3edfe10b0d82785e83bdcb973e22e",
+          "pooling": "useModel",
```

Comment:
since the dump was generated right before importing, its pooling is the newer `useModel`.